### PR TITLE
* add date range picker to reporting in admin-front

### PR DESCRIPTION
**Pull request by JPruefer**
* write tests

### CODE REVIEW

File: admin-front/src/components/Activities/DatetTimePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
```
Comment (RKlompUU) [inline admin-front/src/components/Activities/DatetTimePicker.tsx:1]: There is a typo in the name of this .tsx file (Datet => Date)

File: admin-front/src/data/dataProvider.ts [hunk 2]
```diff
@@ -49,31 +49,33 @@ const dataProvider = (
...
+    switch (resource) {
+      case 'analytics/sales/priceVolume/snapshot':
+        query = {
+          sort: JSON.stringify([
+            field.startsWith('attributes.')
+              ? field.slice('attributes.'.length)
+              : field,
+            order.toLowerCase(),
+          ]),
+          range: JSON.stringify([rangeStart, rangeEnd]),
```
Comment (RKlompUU) [inline admin-front/src/data/dataProvider.ts:61]: Not from this PR itself (this code was already there before), but `snapshot` doesn't have `range` or `sort` logic. Or maybe I am missing something, do you know more about this? If not I'll have a further look at this code to see whether this actually makes sense.

**Review by RKlompUU — APPROVED** [review]
Looks good

File: admin-front/src/data/dataProvider.ts (hunk 2 above)
Comment (JPruefer) [reply admin-front/src/data/dataProvider.ts:61]: Yes, no range or sort logic. Will remove it in the next PR as I do not know if it has any effect on the client code if I remove it from the data provider.

File: admin-front/src/components/Activities/DatetTimePicker.tsx (hunk 1 above)
Comment (JPruefer) [reply admin-front/src/components/Activities/DatetTimePicker.tsx]: Renamed, thanks

**Pull request merged**